contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -2540,6 +2540,143 @@ CREATE OR REPLACE FUNCTION sys.json_query(json_string text, path text default '$
 RETURNS sys.NVARCHAR
 AS 'babelfishpg_tsql', 'tsql_json_query' LANGUAGE C IMMUTABLE PARALLEL SAFE;
 
+-- Function to convert the path format into expected jsonb_set path format using Regular Expression
+CREATE OR REPLACE FUNCTION sys.regex_replace(json_path TEXT)
```

Comment:
maybe a more descriptive name than just `regex_replace()`? Something that describes what the function does instead of how it works, so something like `convert_jsonpath()` or `convert_jsonpath_to_jsonb_path()`

Reply:
Have changed the function name to make it more specific sys.convert_jsonpath_to_jsonb_path